testutil/testkeyring/gen_accounts/gen.go

```diff
@@ -0,0 +1,91 @@
+//go:build ignore
+
+package main
+
+import (
+	"bytes"
+	"encoding/base64"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/cosmos/cosmos-sdk/crypto/hd"
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	"github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/pokt-network/poktroll/testutil/testkeyring"
+
+	"github.com/pokt-network/poktroll/app"
+)
+
+var (
+	flagOut           string
+	flagAccountsLimit int
+	defaultOutPath    = "accounts_table.go"
+)
+
+func init() {
+	flag.StringVar(&flagOut, "out", defaultOutPath, "the path to the generated file.")
+	flag.IntVar(&flagAccountsLimit, "limit", 100, "the number of accounts to generate.")
```

Comment:
Thoughts on s/limit/count or s/limit/n?

I always think of `limit` as `<=`, not `==`

Reply:
I could imagine both `count` or `n` describing a boundary or a counter. I would agree with the argument  that it wouldn't really make much sense for it to be the latter in this case; nonetheless, I thought the word `limit` would sufficiently disambiguate the intended usage.

Reply:
Seems like you did think this through so not strong opinion either way. I think keeping as `limit` is fine.